client/client.go

```diff
@@ -20,17 +20,30 @@ import (
 	"github.com/ShowMax/go-fqdn"
 	"github.com/go-kit/kit/log"
 	"github.com/go-kit/kit/log/level"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 	"github.com/prometheus/common/promlog"
 	"github.com/prometheus/common/promlog/flag"
 	"github.com/robustperception/pushprox/util"
 )
 
 var (
-	myFqdn     = kingpin.Flag("fqdn", "FQDN to register with").Default(fqdn.Get()).String()
-	proxyURL   = kingpin.Flag("proxy-url", "Push proxy to talk to.").Required().String()
-	caCertFile = kingpin.Flag("tls.cacert", "<file> CA certificate to verify peer against").String()
-	tlsCert    = kingpin.Flag("tls.cert", "<cert> Client certificate file").String()
-	tlsKey     = kingpin.Flag("tls.key", "<key> Private key file").String()
+	myFqdn      = kingpin.Flag("fqdn", "FQDN to register with").Default(fqdn.Get()).String()
+	proxyURL    = kingpin.Flag("proxy-url", "Push proxy to talk to.").Required().String()
+	caCertFile  = kingpin.Flag("tls.cacert", "<file> CA certificate to verify peer against").String()
+	tlsCert     = kingpin.Flag("tls.cert", "<cert> Client certificate file").String()
+	tlsKey      = kingpin.Flag("tls.key", "<key> Private key file").String()
+	metricsAddr = kingpin.Flag("metrics-addr", "the").Default(":9514").String()
```

Comment:
9514 is already taken, there's a gap at 9369: https://github.com/prometheus/prometheus/wiki/Default-port-allocations